06_gradeUpdatecode.js

```diff
@@ -0,0 +1,39 @@
+var grades = function(marks) {
```

Comment:
Add some method comments here. So that other people can understand.

Reply:
I have added changes as per comments